tests/unit/test_column_noise.py

```diff
@@ -639,9 +639,83 @@ def test_use_fake_name(dummy_dataset):
     assert noised_last_names.loc[noised_last_names != last_name_data].isin(fake_last).all()
 
 
-@pytest.mark.skip(reason="TODO")
-def test_generate_phonetic_errors():
-    pass
+@pytest.mark.parametrize(
+    "column",
+    [
+        "first_name",
+        "last_name",
+    ],
+)
+def test_generate_phonetic_errors(dummy_dataset, column):
+    data = dummy_dataset[column]
+
+    config = get_configuration()
```

Comment:
Is there a reason you are updating the config on the side? I think it's fine since it's not an integration test but it is skipping all of the validation.

Reply:
Oh I had used code but this is using "real" columns.  I'll change it.